package.json

```diff
@@ -1,24 +1,23 @@
 {
-  "name": "cloudwatch-logs-management",
-  "version": "1.0.0",
+  "name": "@guardian/cloudwatch-logs-management",
```

Comment:
The `package.json` at the root level contains all common dependencies, e.g. typescript, eslint, prettier.